code/datums/mind.dm

```diff
@@ -1718,8 +1718,7 @@
 			if(istype(S, type))
 				continue
 		S.charge_counter = delay
-		spawn(0)
-			S.start_recharge()
+		addtimer(S, "start_recharge", 0)

```

Comment:
Spawns like these should be replaced with set waitfor = 0 procs, as the spawn is about avoiding the sleep in the proc its calling